app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -26,7 +26,17 @@ angular.module("openshiftConsole")
         }];
         scope.claim.selectedLabels = [];
         DataService.list({group: 'storage.k8s.io', resource: 'storageclasses'}, {}, function(storageClasses) {
-          scope.storageClasses = storageClasses.by('metadata.name');
+          var noclass = {
+            metadata: {
+              name: "No Storage Class",
+              labels: {},
+              annotations: {
+                description: "No storage class will be assigned unless a default class has been assigned by the system administrator."
+                }
+            }
+          };
```

Comment:
@spadgett - how do I create a none option in the drop down.  Currently, after you select an option, you cannot unselect.  This why I added the "No Storage Class" option.  I would like to remove it if the user can choose blank or none in the drop down.